react/components/ProductImages/components/ProductImage.tsx

```diff
@@ -71,7 +71,7 @@ const ProductImage: FC<Props> = ({
                 MAX_SIZE,
                 aspectRatio
               )}
-              className={handles.productImageTag}
+              className={`${handles.productImageTag} ${handles.productImageZoom}`}
```

Comment:
Can you use the `applyModifier` helper instead of adding another handle? 

```
import { applyModifiers } from 'vtex.css-handles'
...
className={`${applyModifiers(handles.productImageTag, 'zoom')}`}
```

Would result in `productImageTag` and `productImageTag--zoom` 😁 

https://github.com/vtex-apps/css-handles#usecsshandles

Reply:
@kaisermann I made the correction.